### PR TITLE
Update Vagrant provisioning for Crane with Docker v2.

### PR DESCRIPTION
**Pull request by rbarlow**

### CODE REVIEW

File: playpen/vagrant-setup.sh [hunk 1]
```diff
@@ -47,8 +47,10 @@ debug: true
 endpoint: pulp-devel:5001
 EOF
 
-    mkdir -p metadata
-    sudo ln -s $HOME/devel/crane/metadata/ /var/lib/pulp/published/docker/app
+    mkdir -p metadata/v1 metadata/v2
+    sudo mkdir -p /var/lib/pulp/published/docker/v1 /var/lib/pulp/published/docker/v2
```
Comment (ipanova) [inline playpen/vagrant-setup.sh]: is not it supposed to be '/var/lib/pulp/published/docker/v1/app'? same for v2 version.

Comment (rbarlow) [inline playpen/vagrant-setup.sh]: I don't believe so, because the two lines below this create "app" as a symlink.

**Comment by rbarlow** [comment]
@ipanova Can I get an LGTM?

**Comment by ipanova** [comment]
ok test

**Pull request merged**